### DISCUSSION TITLE
config.txt setting to start with "Advanced" view open by default?

### DISCUSSION THREAD

**Discussion by JRice** [lllyasviel/Fooocus#2707] [Q&A] [answered]
...Is it possible to have Fooocus start with the "advanced" option "open" (the options at the right viewable by default)?

**Comment by DarkNeo88** [comment, marked as answer]
read config_modification_tutorial.txt
"default_advanced_checkbox": true

**Comment by JRice** [comment]
Ahhh, indeed. Thanks.

I did, of course, read the tutorial, but as nothing is _explained_, I missed that one. I appreciate the pointer.
